app/static/tests/e2e/options.etest.ts

```diff
@@ -434,20 +434,23 @@ test.describe("Options Tab tests", () => {
         await editDisplayName(2, page);
         await inputDisplayName(2, page, "random name 1");
         await saveDisplayName(1, page);
-        await expect((await page.innerText(":nth-match(.tooltip-inner, 2)")).trim()).toBe("Name already exists");
-        await expect(await page.isVisible(":nth-match(.param-name-input, 2)")).toBe(true);
+        await expect(await page.getByText("Name already exists")).toBeVisible();
     });
 
     test("error tooltip and doesn't change name if Set [number] format used", async ({ page }) => {
         await createParameterSet(page);
         await editDisplayName(1, page);
         await inputDisplayName(1, page, "Set 10");
         await saveDisplayName(1, page);
-        await expect((await page.innerText(":nth-match(.tooltip-inner, 2)")).trim()).toBe(
+        await expect(await page.getByText(
             "Set 10 (or any Set [number] combination) is reserved for default set names. " +
-                "Please choose another set name or name this set back to its original name of 'Set 1'"
-        );
-        await expect(await page.isVisible(".param-name-input")).toBe(true);
+            "Please choose another set name or name this set back to its original name of 'Set 1'"
+        )).toBeVisible();
+        // await expect((await page.innerText(":nth-match(.tooltip-inner, 2)")).trim()).toBe(
+        //     "Set 10 (or any Set [number] combination) is reserved for default set names. " +
+        //         "Please choose another set name or name this set back to its original name of 'Set 1'"
+        // );
+        // await expect(await page.isVisible(".param-name-input")).toBe(true);
```

Comment:
This can be removed now?

Reply:
yep removed in the linting PR (next one)